spring-cloud-kubernetes-fabric8-leader/src/test/java/org/springframework/cloud/kubernetes/fabric8/leader/LeaderInitiatorTest.java

```diff
@@ -83,7 +83,9 @@ public void shouldStart() throws InterruptedException {
 		assertThat(this.leaderInitiator.isRunning()).isTrue();
 		verify(this.mockFabric8LeaderRecordWatcher).start();
 		verify(this.mockFabric8PodReadinessWatcher).start();
-		Thread.sleep(10);
+
+		// TODO this tests needs to be reviewed not to use sleep
```

Comment:
otherwise github maven is failing, but overall this needs to be re-written in the long run.